docs/source/api.rst

```diff
@@ -154,3 +165,4 @@ Return Iterator Types
    retworkx.NodeIndices
    retworkx.EdgeList
    retworkx.WeightedEdgeList
+   retworkx.Pos2DMapping
```

Comment:
with `retworkx.*_layout(graph).keys()` a user will see `Pos2DMappingKeys`. similar with`.values()`. Should we have docs for these types?

Reply:
I'm not sure if it matters too much, we can add it to the documentation it doesn't hurt. I just left it out since there really isn't any documentation for those types and we don't actually export them in the retworkx module. I basically modeled these types on how python's dict works it 'returns a `dict_keys`, `dict_values`, and `dict_items` class for `.keys()`, `.values()`, and `.items()`. I couldn't find any documentation on those classes except in the C-API: https://docs.python.org/3/c-api/dict.html?highlight=dict_keys#c.PyDict_Keys